src/systemdhook.c

```diff
@@ -822,8 +822,14 @@ int main(int argc, char *argv[])
 		snprintf(config_file_name, PATH_MAX, "%s/config.json", YAJL_GET_STRING(v_bundle_path));
 		fp = fopen(config_file_name, "r");
 	} else {
-		char msg[] = "bundlePath not found in state";
+		/****
+		* On Docker versions prior to 1.12, bundlePath will not
+		* be provided.  Let's exit quietly if not found.
+		****/
+		char msg[] = "bundlePath not found";
 		snprintf(config_file_name, PATH_MAX, "%s", msg);
+		pr_pinfo("Failed reading state data: %s.  Generally this indicates Docker versions prior to 1.12 are installed.", config_file_name);
+		return EXIT_SUCCESS;
 	}
```

Comment:
It took a while to get this patch out as I just discovered that the "Failed reading state..." message is now printed twice and I"m not sure why.  It's printed when the issue is first encountered as the container is being started and then secondarily after the container is closed/exited/shutdown.  I think it's an existing issue with pr_pinfo, but wanted to point that out.